src/components/Calculator.js

```diff
@@ -0,0 +1,30 @@
+export default function Calculator() {
+  return (
+    <>
+      <div className="container">
+        <input type="text" name="calc" className="input" />
+        <div className="itemsContainer">
+          <button type="button" className="items">+/-</button>
+          <button type="button" className="items">%</button>
+          <button type="button" className="items">AC</button>
+          <button type="button" className="items orange">/</button>
```

Comment:
- [ ] Kindly use the same division symbol (&div;) we have on the design. It's important that we stick to the design